docs/content/configuration/sumdb.md

```diff
@@ -5,41 +5,47 @@ weight: 2
 
-So for example, if you run the following command: 
+So for example, if you run the following command:
 
 ```bash
 GOPROXY=<athens-url> go build
 ```
 
-The Go command will proxy requests to `sum.golang.org` like this: `<athens-url>/sumdb/sum.golang.org`. Feel free to read the linked proposal above for the exact requests that makes Athens successfully proxy Checksum DB APIs. 
+The Go command will proxy requests to `sum.golang.org` like this: `<athens-url>/sumdb/sum.golang.org`. Feel free to read the linked proposal above for the exact requests that makes Athens successfully proxy Checksum DB APIs.
 
 Note that as of this documentation (May 2019), you need to explicitly set `GOSUMDB=https://sum.golang.org`, but the Go team is planning on enabling this by default.
 
-### Why a Checksum DB? 
-
-The reasons for needing a Checksum DB is explained in the linked proposal above. However, the reasons for proxying a Checksum DB are more explained below. 
-
-### Why Proxy a Checksum DB? 
+### Why Proxy a Checksum DB?
 
 This is quite important. Say you are a company that is running an Athens instance, and you don't want the world to know about where your 
 repositories live. For example, say you have a private repo under `github.com/mycompany/secret-repo`. In order to ensure that the Go client 
-does not send a request to `https://sum.golang.org/lookup/github.com/mycompany/secret-repo@v1.0.0` and therefore leaking your private import path to the public, you need to ensure that you tell Go to skip particular import paths as such: 
+does not send a request to `https://sum.golang.org/lookup/github.com/mycompany/secret-repo@v1.0.0` and therefore leaking your private import path to the public, you need to ensure that you tell Go to skip particular import paths by setting the `GONOSUMDB` environment variable:
 
 ```
 GONOSUMDB=github.com/mycompany/* go build
 ```
 
-This will make sure that Go does not send any requests to the Checksum DB for your private import paths. 
-However, how can you ensure that all of your employees are building private code with the right configuration? 
+This will make sure that Go does not send any requests to the Checksum DB for your private import paths. However, how can you ensure that all of your employees are building private code with the right configuration?
+
+Athens, in this case can help ensure that all private code flowing through it is never checked against the checksum DB. That means that as long as your employees are using Athens, then they will get a helpful reminder to ensure Their GONOSUMDB is rightly configured.
 
-Athens, in this case can help ensure that all private code flowing through it never goes to the Checksum DB. So as long as your employees are using Athens, then they will get a helpful reminder to ensure Their GONOSUMDB is rightly configured. 
+As the Athens company maintainer, you can run Athens with the following configuration in your `config.toml` file:
```

Comment:
"Athens company maintainer" seems odd here, as it may not always be a company running  Athens. Let's just keep this simple: 
```suggestion
Athens allows the user to specify a list of patterns to not lookup via the upstream sum provider:
```

Reply:
Agree with @twexler , although an "Athens user" is a bit ambiguous because it can mean one of two things: 
1. Athens server maintainer (the person who configures and runs Athens)
2. Athens server user (the person pinging Athens directly or through the Go command) 

So maybe we can just say "Athens allows its operator to specify a list of patterns..."
or "Athens allows its maintainer to specify a list of patterns..."

Reply:
I applied @twexler's commit. I'll change it to add the "operator" qualifier